crates/nu-cli/src/completions/variable_completions.rs

```diff
@@ -44,9 +44,7 @@ impl Completer for VariableCompletion {
     ) -> Vec<Suggestion> {
         let mut output = vec![];
         let builtins = ["$nu", "$in", "$env"];
-        let var_str = std::str::from_utf8(&self.var_context.0)
-            .unwrap_or("")
-            .to_lowercase();
+        let var_str = std::str::from_utf8(&self.var_context.0).unwrap_or("");
```

Comment:
Am I reading this correctly that the variable completions will become case sensitive?

Reply:
I believe `var_str` is only used to compare against `$nu` and `$env`. That comparison is now case sensitive, but when I tested I wasn't getting completions on `$Env.<tab>` anyway. This one could go either way tbh.

Reply:
Ah this is the logic for the subpaths of those. Can confirm your observation and it should still handle `$E<tab>` to `$env` etc. 

I am fine with that.